operators/go/gen_subm_operator.sh

```diff
@@ -111,10 +111,7 @@ function add_subm_engine_to_operator() {
 
   operator-sdk add controller --api-version=$api_version --kind=$kind
 
-  # FIXME: This needs to point at the under-dev/un-merged custom controller file
-  #controller_file_src=$GOPATH/src/github.com/submariner-io/submariner/operators/go/submariner_controller.go
-  # FIXME: Don't have such a hard assumption about the location of custom controller
-  controller_file_src=$HOME/submariner/operators/go/submariner_controller.go
+  controller_file_src=$GOPATH/src/github.com/submariner-io/submariner/operators/go/submariner_controller.go
```

Comment:
Maybe I'm confused, but I'm not sure seeing how this works. How does the under-dev controller src get to this path?

Reply:
Oh, are you basically saying this is the correct location to keep the under-dev src? That makes sense, I just need to change where I have my repo.

Reply:
Yes, the expectation seems to be that people develop under `$GOPATH/src`.